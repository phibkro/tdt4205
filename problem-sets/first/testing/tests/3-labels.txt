8: go
10: dx=1
-11: go go
100: go dx=5
apple: go
go: go
