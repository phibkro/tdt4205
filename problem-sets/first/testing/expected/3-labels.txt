Table filled!
line    1: accepted: 8: go
line    2: accepted: 10: dx=1
line    3: error: -11: go go
~~~~~~~~~~~~~~~~~~^
line    4: accepted: 100: go dx=5
line    5: error: apple: go
~~~~~~~~~~~~~~~~~~^
line    6: error: go: go
~~~~~~~~~~~~~~~~~~~~^
